File: lc1550.java
public class lc1550 {
//    1550. Three Consecutive Odds
    public static void main(String[] args) {
        int arr[]={1,2,34,3,4,5,7,23,12};
        boolean ans=false;
        if (arr.length>=3){
            for (int i=0;i<arr.length-2;i++){
                if (arr[i]%2!=0 && arr[i+1]%2!=0 && arr[i+2]%2!=0){
                    ans=true;
                    break;
                }
            }
        }
        System.out.println(ans);
    }

}

File: lc1550.py
from typing import List


class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        ans=False
        for i in range(0,len(arr)-2):
            if (arr[i]%2!=0) & (arr[i+1]%2!=0) & (arr[i+2]%2!=0):
                ans=True
                break
        return ans
print(Solution().threeConsecutiveOdds([1,2,3,5,7]))